### ISSUE TITLE
Proposal to keep communications more open

### ISSUE THREAD

**Issue by koresar** [nodejs/board#70]
Hello.

[Yesterday](https://github.com/nodejs/board/issues/67#issuecomment-327641411) board came up with several good changes like moderation etc. Also, another good intent, quoting:
> The Board will continue to work with the community to find a constructive way forward and to improve the governance...

Here is an idea on improving governance.

Looking at the latest events happened in the Foundation I came to the conclusion that **closed communications is the root cause** of the recent CoC-related issues.

Apache Software Foundation have [this rule](https://www.apache.org/foundation/how-it-works.html#confidential) about community communications:

> ### BALANCING CONFIDENTIALITY AND PUBLIC DISCUSSION
> 
> **We endeavour to conduct as much discussion in public as possible.** This encourages openness, provides a public record, and stimulates the broader community.
> 
> However sometimes internal private mail lists are necessary. You must never divulge such information in public without the express permission of the list. Also never copy an email between private and public lists (no Cc). Such an event would go beyond the normal need for email ettiquette and be a serious breach of confidence. It could have serious ramifications, cause unnecessary confusion and ill-informed discussion.
> 
> **Private lists are typically only used for matters pertaining to people as individuals (like voting in new committers), and legal matters that require confidentiality.**

------

My proposal to Node.js Foundation - copy the best. Apply that rule.

Open PR reviews, open all possible issues on GitHub, discourage closed communications, open committee discussions, open everything you can. Limiting those who can speak is fine, but hiding information will definitely cause another CoC misconduct or other widely resonating problems.

_PS: my English could be much better, it's hard to explain things for me using foreign language, please understand that the text above could be beautifully written, I just can't._

**Comment by Morgul** [comment]
@koresar Your English is fine, and better than some native speakers I know. :)

I love this idea. That it's also a core rule of the Apache Foundation is adding to what I feel is a very reasonable proposal. Of the communities I'm a member of, I'm always much more active in the ones where it's easy to find out what's going on, see what the leaders are discussing, and that I just feel more connected with.

The practical question here, then, would be what actual changes would adopting this rule bring? I'm not sure. Maybe someone on the board has some ideas?

**Comment by jakeNiemiec** [comment]
@koresar I totally agree that this would play a large role in restoring confidence in leadership. 

note: You may need to post this in @nodejs/tsc instead. I would ask someone like @hackygolucky, what the protocol for a proposal like this is. PR? Issue?

**Comment by MylesBorins** [comment]
One thing I would like to add.

I think clear and open communication is extremely important. That being
said I do not think it is appropriate for CoC related discussions to happen
in the open.

It is unfair to the reporters and to those who are facing claims for the
grievences to happen openly.

All of our working group meetings do happen on air. The videos are recorded
and available on YouTube. Further the notes are all published to github

The only communications that have been private in the committees are those
involving individuals and those involving security. All of which falls into
line with the Apache rule provided.

I disagreed that the problems we are seeing are due to private discussions,
but rather due to information being published regarding an individual that
should have been anonymized.

I completely empathize with how it can feel like things have been less than
public, and an open to specific suggestions on how we can improve.

We have discussed created a more explicit process for making complaints
against committee members made public, anonymized unless there is express
concent from those involved. Would a policy like that satisfy your concerns?

**Comment by williamkapke** [comment]
@koresar I am working to improve transparency between the Node Foundation Board and the community- but it takes time. It really helps for folks, like yourself, to drop in and voice these opinions and show your support- so: Thank you!

It isn't too obvious to the community that the Node.js Foundation Board doesn't strive to be involved with the day-to-day decision making (including CoC topics). One of the goals we hope to achieve is to make it so the Board has **almost nothing** to discuss- and therefor, nothing hidden. This is done by setting up and empowering **Committees that own all decision making**. The [Community Committee](https://github.com/nodejs/community-committee/) is the latest example of this process in action.

...and, as @MylesBorins noted- our committees have an impeccable track record for operating transparently just as you suggest:
> Open PR reviews, open all possible issues on GitHub, discourage closed communications, open committee discussions, open everything you can.

BUT, the Foundation is still fairly new. We're still figuring out what Committees deal with what ...and sometimes DO end up having more in-depth conversations. Many believe there is value in the community knowing that the Board has been informed of topics/issues and are on standby to provide resources if needed. We have not done a great job communicating these things though. For legal reasons- communicating things at the board level needs to be more formal than it is for Committees.

... but we're making progress! The post from @mrhinkle is proof of this.
- **This repo is new**<sup>ish</sup>: When I was elected, I pushed to get this repo created. I felt it was a simple, yet vital, basic need to facilitate communication for board topics.
- **@mrhinkle is new**<sup>ish</sup>: He is doing or Executive Director work now. This means we actually have someone assigned/authorized to give updates! We've never had that before! Now we can acclimate ourselves to having this new resource, iterate, and improve.

As to your proposal for having something similar to Apache. We do have a "Guiding Principle" note at the beginning of the carters for the TSC and CommComm:
* https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-1-guiding-principle
* https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-1-guiding-principle

I think Apache's statement is a bit more thorough/concrete however and welcome PRs proposing improvements ours. Perhaps, something new that isn't in a committee charter and applies to the entire Foundation.

**Comment by jakeNiemiec** [comment]
### 1. Private vs published information
>I disagreed that the problems we are seeing are due to private discussions,
but rather due to information being published regarding an individual that
should have been anonymized.

@MylesBorins   **I agree** with what you are trying to say here, but with one correction: It was due to **vague accusations** being published regarding an individual **without any additional information**. 

The public needs **to be able to sympathize with the ACTION of a moderator standing up for the transgressed**. Both victim and accused are out of scope and should remain anonymous. The CoC covers everything from `a clear and imminent threat to life` to `insensitive speech`. Accusing a (even anonymous) member of committing "something within that range" is [unacceptable](https://www.youtube.com/watch?v=07So_lJQyqw).

Suggestions on this point:

1. [Sectionize the CoC](https://www.law.cornell.edu/uscode/text/17/107) so that you can scope the severity of the violation.
2. Be transparent about the steps you are taking to resolve the situation without sharing details. E.g: `We are currently collecting information from both parties` or `We are currently in mediation` or `The matter has been resolved and the accused has been reprimanded`. 
3. Don't leave it up to imagination. All you are doing is feeding the biases people already have. It does nothing but fuel the social media fire🔥.

### 2. Throwing gas on a dumpster fire

As you said in _An alternative strategy to moderation_ https://github.com/nodejs/community-committee/issues/122 (others please read for full context):

> TLDR; we should not be codifying specific acts but rather **rely on individual contributors reaction on a case by case basis** and we should be **more liberal with our banning**. The most important thing is to make a safe space for individuals to contribute. The second most important thing is to **avoid derailment**.

This was such a: `YES..wait...noo` moment for me. And it just keeps getting worse with your statement above:

>I do not think it is appropriate for CoC related discussions to happen
in the open.

Please correct me if you didn't mean `discussions`, but rather meant `arbitration`. 

### 3. Damage done
And finally we arrive at your comment above: 
>**The only communications that have been private in the committees are those
involving individuals** and those involving security. All of which falls into
line with the Apache rule provided.

This could not be further from reality. As of the time this is written, the discussion detailing and surrounding `the accused's` CoC violations now outranks the **_actual CoC_**! _(for the google search `Nodejs CoC` and others)_

![image](https://user-images.githubusercontent.com/16782063/30236286-e144d05a-94db-11e7-8d32-2a23161c8bae.png)

The worst thing is:
### Anyone searching something like `Nodejs CoC` in the foreseeable future will be faced with this lasting image of a bitter community in turmoil.

**Comment by koresar** [comment]
Last time a spoke with @brettporter (was chairman of ASF for >6 years) about why ASF is so successful he listed few main rules of the Apache governance. IIRC the first one he mentioned was that "strict rule is to always communicate on public". (Brett, if you are reading this please confirm.)

Please, find the PR https://github.com/nodejs/TSC/pull/391 I have just created. This is almost a copycat of the [ASF rule](https://www.apache.org/foundation/how-it-works.html#confidential).
Wording is probably somewhat off and better be revised, but please see the intent behind the idea, not the words. (Because I usually can't come up with the right English wording. Sorry.)

**Comment by Trott** [comment]
Closing stale issues and PRs as the repository is being archived.
